tests/UI/specs/Dashboard_spec.js

```diff
@@ -223,4 +223,14 @@ describe("Dashboard", function () {
         }, done);
     });
 
+    it("should load correctly with token_auth", function (done) {
+        testEnvironment.testUseMockAuth = 0;
+        testEnvironment.save();
+
+        expect.screenshot("loaded_token_auth").to.be.capture(function (page) {
```

Comment:
Guess that will only show the login form as the initial request will "fail". But I'll add one.